webui/src/Controls/InternalModuleField.tsx

```diff
@@ -225,6 +227,28 @@ export const InternalPageIdDropdown = observer(function InternalPageDropdown({
 	}
 })
 
+// Formatting for variable pulldown options:
+const CustomOption = React.memo((props: OptionProps<DropdownChoiceInt>) => {
```

Comment:
As these are similar (identical?) to the ones for the text input dropdown, perhaps it should be the same component in a 'DropdownUtil.tsx'?
There might be some differences though which makes this not viable.

Reply:
Done. Combined as described in  my response to your comment on line R28.